package.json

```diff
@@ -95,5 +62,29 @@
       "path": "./node_modules/cz-conventional-changelog"
     }
   },
-  "version": "0.0.0-development"
+  "jest": {
+    "testEnvironment": "node",
+    "setupFiles": [
+      "./testSetup.js"
+    ]
+  },
+  "greenkeeper": {
+    "ignore": [
+      "cosmiconfig"
+    ]
+  },
+  "version": "0.0.0-development",
```

Comment:
I'd move it to the top to make it very explicit that we use semantic-release.

Reply:
In which line number would you want it? after `name`?

Reply:
Yeah, I think it's better that way